app/src/organisms/ProtocolSetupParameters/ViewOnlyParameters.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  ALIGN_CENTER,
+  BORDERS,
+  COLORS,
+  DIRECTION_COLUMN,
+  DIRECTION_ROW,
+  Flex,
+  SPACING,
+  TYPOGRAPHY,
+} from '@opentrons/components'
+import { useMostRecentCompletedAnalysis } from '../LabwarePositionCheck/useMostRecentCompletedAnalysis'
+import { ChildNavigation } from '../ChildNavigation'
+import { StyledText } from '../../atoms/text'
+import { Chip } from '../../atoms/Chip'
+import { useToaster } from '../ToasterOven'
+import { mockData } from './index'
+
+import type { RunTimeParameter } from '@opentrons/shared-data'
+import type { SetupScreens } from '../../pages/ProtocolSetup'
+
+export interface ViewOnlyParametersProps {
+  runId: string
+  setSetupScreen: React.Dispatch<React.SetStateAction<SetupScreens>>
+}
+
+export function ViewOnlyParameters({
+  runId,
+  setSetupScreen,
+}: ViewOnlyParametersProps): JSX.Element {
+  const { t, i18n } = useTranslation('protocol_setup')
+  const { makeSnackbar } = useToaster()
+  const mostRecentAnalysis = useMostRecentCompletedAnalysis(runId)
+  const makeSnack = (): void => {
+    makeSnackbar(t('reset_setup'))
+  }
+
+  //  TODO(jr, 3/18/24): remove mockData
+  const parameters = mostRecentAnalysis?.runTimeParameters ?? mockData
+
+  const getDefault = (parameter: RunTimeParameter): string => {
+    const { type, default: defaultValue } = parameter
+    const suffix =
+      'suffix' in parameter && parameter.suffix != null ? parameter.suffix : ''
+    switch (type) {
+      case 'int':
+      case 'float':
+        return `${defaultValue.toString()} ${suffix}`
+      case 'boolean':
+        return Boolean(defaultValue)
+          ? i18n.format(t('on'), 'capitalize')
+          : i18n.format(t('off'), 'capitalize')
+      case 'str':
+        if ('choices' in parameter && parameter.choices != null) {
+          const choice = parameter.choices.find(
+            choice => choice.value === defaultValue
+          )
+          if (choice != null) {
+            return choice.displayName
+          }
+        }
+        break
+    }
+    return ''
+  }
+
+  return (
+    <>
+      <ChildNavigation
+        header={t('parameters')}
+        onClickBack={() => setSetupScreen('prepare to run')}
+        chipProps={{
+          type: 'info',
+          iconName: 'information',
+          text: t('values_are_view_only'),
+        }}
+      />
+      <Flex
+        marginTop="7.75rem"
+        flexDirection={DIRECTION_COLUMN}
+        gridGap={SPACING.spacing8}
+        paddingX={SPACING.spacing8}
+      >
+        <Flex
+          gridGap={SPACING.spacing8}
+          color={COLORS.grey60}
+          fontSize={TYPOGRAPHY.fontSize22}
+          fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+          lineHeight={TYPOGRAPHY.lineHeight28}
```

Comment:
from Figma

<img width="730" alt="Screenshot 2024-03-20 at 3 10 09 PM" src="https://github.com/Opentrons/opentrons/assets/474225/23209feb-5954-4d40-893b-a3ea3466e329">


```suggestion
          fontSize={TYPOGRAPHY.fontSize20}
          fontWeight={TYPOGRAPHY.fontWeightSemiBold}
          lineHeight={TYPOGRAPHY.lineHeight24}
```